proposals/lip-0050.md

```diff
@@ -90,24 +99,55 @@ This substore contains an entry for each legacy address for which no [reclaim tr
 
 Obtaining the legacy 8-byte address from a public key.
 
-##### Parameters
-
-A 32-byte value representing a public key.
-
 ##### Returns
 
 The 8-byte legacy address corresponding to the given input.
 
 ##### Execution
 
 ```python
-getLegacyAddress(publicKey):
+def getLegacyAddress(publicKey: PublicKeyEd25519) -> bytes:
     hashedKey = SHA-256(publicKey)
     firstEightBytes = first 8 bytes of hashedKey
     reversedEightBytes = firstEightBytes reversed
     return reversedEightBytes
 ```
 
+### Events
+
+#### AccountReclaimed
+
+This event has `typeID = TYPE_ID_ACCOUNT_RECLAIM`.
+This event is emitted when a legacy account is reclaimed.
+
+##### Topics
+
+* `legacyAddress`: the legacy address of the reclaimed account.
+* `address`: the address of the reclaimed account.
+
+##### Data
+
+```java
+accountReclaimedEventDataSchema = {
+    "type": "object",
+    "required" = ["legacyAddress", "address", "amount"],
+    "properties": {
+        "legacyAddress": {
+            "dataType": "bytes",
+            "fieldNumber": 1
+        },
+        "address": {
+            "dataType": "bytes",
+            "fieldNumber": 2
+        },
+        "amount": {
+            "dataType": "uint64",
+            "fieldNumber": 3
+        },
+    }
+}
+```
+
 ### Commands
 
 #### Reclaim
```

Comment:
As we wanted to use this PR to align on module LIPs, some style comments:

- I think it would be sufficient to write "This command has module ID `COMMAND_ID_RECLAIM`" as the module ID is clear from the context.
- I think the sentence "The `params` property of a reclaim transaction must obey the following schema:" can be removed.

Reply:
Did we align on this? I would say that this is a level of detail that can be left to the LIP author and does not need to be uniform.

Reply:
Nothing mandatory, just optional suggestions. I resolve it.